src/index.ts

```diff
@@ -29,6 +34,12 @@ export interface PersistedStateOptions {
    */
   overwrite?: boolean
 
+  /**
+   * Serializer to use
+   * @default { serialize: JSON.stringify, deserialize: JSON.parse }
+   */
+  serialize?: Serializer
```

Comment:
expose both functions instead of an interface

Reply:
The reason I did it this way is because if you have a _custom serializer_ your deserializer probably won't be `JSON.parse`!

Reply:
I'm fine with `Serializer` interface, and exposing the whole interface as optional option with both functions needed, maybe call the option `serializer`.

```ts
serializer?: Serializer
```

Reply:
I wanted to change the name anyway 😁